force_bdss/mco/optimizer_engines/space_sampling.py

```diff
@@ -40,12 +40,6 @@ class SpaceSampler(ABCHasStrictTraits):
     def __init__(self, dimension, resolution, **kwargs):
         super().__init__(dimension=dimension, resolution=resolution, **kwargs)
 
-    @abc.abstractmethod
-    def _get_sample_point(self):
-        """ Internal generator of sample points from the distribution.
-        Returns or yields a sampled point.
-        """
-
```

Comment:
I didn't make any changes to this module and I 'm pretty sure I pulled from master, so I'm not sure what's going on here. But it doesn't make any difference to running.

Reply:
Yes, the diff will change if any code is moved around, the functionality looks the same